lib/fibonacci.rb

```diff
@@ -1,20 +1,11 @@
 
 class Fibonacci
```

Comment:
I think that the class would be very useful if you've decided to extend the functionality to return the sequence too. You could then have two functions in `Fibonacci`, `fib_number` and `fib_sequence`. With only `fib` (fib number), the class wasn't necessary, but at the same time it's not "wrong" having it :)